### PR TITLE
Updated Dockerfile to use an Ubuntu image that's not deprecated

### PR DESCRIPTION
**Pull request by carlos-mg89**
HOW TO RUN:

1.  Be sure your Docker containers are down => `docker-compose down`
2.  Perhaps you need to run this as well => `sudo rm -R .tmp/`
3.  Remove your Docker Containers and build them again => `docker/up --build`

### CODE REVIEW

**Comment by carlos-mg89** [comment]
Useful info:

How to upgrade PHP version

- Edit `docker/php/Dockerfile` and change the `PHP_VERSION` variable
- Edit `docker-compose.yml` so we use the right PHP-FPM version.
  - E.g.: `./docker/php/php-ini-overrides.ini:/etc/php/7.1/fpm/conf.d/99-overrides.ini`
- Edit `docker/php/php-fpm-startup` to use the right PHP-FPM version
- Edit `grunt/run.js` so the first value on the `phps` array is the one you chose
- Edit `composer.json` so the min PHP version is the one you want to upgrade to

File: docker/php/Dockerfile [hunk 1]
```diff
@@ -2,28 +2,45 @@
 # PHPDocker.io PHP 7.1 / FPM image #
 ####################################
 
-FROM phpdockerio/php71-cli
+FROM ubuntu:bionic-20210512
```
Comment (davidbeig) [inline docker/php/Dockerfile]: Could we change this to: 
```FROM ubuntu:bionic```?

I don't know for how long ubuntu keeps its images based on the compilation date

**Review by davidbeig — COMMENTED** [review]
Thanks @carlos-mg89.

Great work here! I just left you one comment, and then we can merge!

File: docker/php/Dockerfile (hunk 1 above)
Comment (carlos-mg89) [reply docker/php/Dockerfile]: @davidbeig done! Good advice, didn't notice about this, but I'm sure it's important. I've tested it with ubuntu:bionic and it works OK.

After reading this documentation https://hub.docker.com/_/ubuntu?tab=description&page=1&ordering=last_updated it seems there are more possibilities, like ubuntu:18.04 but I think we're good with this one :)

**Review by davidbeig — APPROVED** [review]
Perfect!

**Pull request merged**